uploaders/xcarchiveuploader.go

```diff
@@ -29,22 +30,29 @@ func DeployXcarchive(pth, buildURL, token string) error {
 		return nil
 	}
 
-	infoPlistPth, err := xcarchive.GetEmbeddedInfoPlistPath(filepath.Join(unzippedPth, pathutil.GetFileName(pth)))
+	appInfoPlistPth, err := xcarchive.GetEmbeddedInfoPlistPath(filepath.Join(unzippedPth, pathutil.GetFileName(pth)))
 	if err != nil {
 		return fmt.Errorf("failed to unwrap Info.plist from xcarchive, error: %s", err)
 	}
 
-	infoPlistData, err := plistutil.NewPlistDataFromFile(infoPlistPth)
+	appInfoPlistData, err := plistutil.NewPlistDataFromFile(appInfoPlistPth)
```

Comment:
NOTE: this functionality could be added to xcarchive package

Reply:
We even have better:
https://github.com/bitrise-io/go-xcode/blob/master/xcarchive/ios.go#L188